spring-boot-starter/external-task-client/loan-granting-spring-boot-webapp/pom.xml

```diff
@@ -85,13 +85,20 @@
       <artifactId>js-scriptengine</artifactId>
       <version>${graaljs.version}</version>
     </dependency>
+
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+      <version>2.3.1</version>
```

Comment:
🔧 The version can be controlled from a property

Reply:
Done